src/redux/reducers.js

```diff
@@ -5,6 +5,7 @@ const {request, response, fail} = ActionHelpers
 
 import _ from 'lodash'
 import {userParser, postParser} from '../utils'
+import {frontendPreferences as frontendPrefsConfig} from '../config'
```

Comment:
this is a bad pattern. I'm not sure that it will work after we migrate to webpack2, which has its own ES6-modules implementation.

config exports only default, it doesn't have named exports and this syntax is for importing named exports.

I guess it is ok for now (and I see that it is used in other parts of the code already)